--- assets/js/main.js
+++ assets/js/main.js
@@ -108,8 +108,10 @@
                                 </div>
                             ` : `
                                 <iframe src="${project.link}embed/?hidecaption=true" frameborder="0" scrolling="no" allowtransparency="true" class="cs-reel-iframe"></iframe>
-                                <div class="cs-click-shield-top"></div>
-                                <div class="cs-click-shield-bottom"></div>
+                                <div class="cs-shield cs-shield-top"></div>
+                                <div class="cs-shield cs-shield-bottom"></div>
+                                <div class="cs-shield cs-shield-left"></div>
+                                <div class="cs-shield cs-shield-right"></div>
                             `}
                         </div>
                         
